### PR TITLE
feat(services): add override catalog

### PR DESCRIPTION
**Pull request by InteractiveTimmy**
### Description

The scope of the changes in this pull request are to add an`override` catalog that supersedes the priority of the `postauth` catalog. This is primarily for improving test flow, but could be used in cases of service outages.

### CODE REVIEW

**Comment by InteractiveTimmy** [comment]
A unit test locally fails that is unrelated to these changes:

```
Credentials
      Token
        #initialize()
          ✖ computes expires_in and refresh_token_expires_in if not specified
```

**Review by mwegman — CHANGES_REQUESTED** [review]
Can you provide integration tests for this?

File: packages/node_modules/@webex/webex-core/src/lib/services/service-catalog.js [hunk 1]
```diff
@@ -14,15 +14,20 @@ const ServiceCatalog = AmpState.extend({
...
+      signin: []
     }))],
     status: ['object', true, (() => ({
       discovery: {
         ready: false,
         collecting: false
       },
+      override: {
+        ready: false,
+        collectiong: false
```
Comment (chrisadubois) [inline packages/node_modules/@webex/webex-core/src/lib/services/service-catalog.js:29]: collecting

Comment (InteractiveTimmy) [reply packages/node_modules/@webex/webex-core/src/lib/services/service-catalog.js]: fixed!

**Comment by InteractiveTimmy** [comment]
> Can you provide integration tests for this?

Added the integration test that covers the entire stack @mwegman

**Review by chrisadubois — APPROVED** [review]

**Comment by adamweeks** [comment]
@mwegman can you re-review or remove your request for changes?

**Comment by chrisadubois** [comment]
Merge it !!!

**Review by mwegman — APPROVED** [review]

**Pull request merged**